### DISCUSSION TITLE
Azure hybrid, PHS a atrybut PwdLastSet

### DISCUSSION THREAD

**Discussion by ghost** [AzureCommunityPL/group#116] [Pomoc]
Hej,
Mamy Hybrydę, wykorzystujemy Password Hash Sync i password writeback.
W lokalnym AD mamy politykę 35 dni ważności hasła ustawioną 15stego dnia miesiąca. W O365 ustawiłem politykę 35 dni ważności hasła 18stego dnia miesiąca.

Co się teraz stanie, kiedy userowi wygaśnie hasło? Czy te daty się zsynchronizują?

**Comment by eklime** [comment]
No ale źródłem tożsamości, a tym samym hasła jest w dalszym ciągu lokalne AD.
Aaa no i... wymuszanie zmiany haseł tak często nie jest już zalecane od dobrych kilku lat - pomyśl o zmianie ;)
